README.md

```diff
@@ -12,21 +12,44 @@ Setting up a fully working Linux system with all needed packages and openHAB rec
 openHABian aims to provide a **self-configuring** Linux system setup specific to the needs of every openHAB user.
 The project provides two things:
 
-*   A set of scripts to set up openHAB on any Debian/Ubuntu based system
-*   Complete **SD-card images pre-configured with openHAB** and many other openHAB- and Hardware-specific preparations, for the *Raspberry Pi* and the *Pine A64* platforms.
+*   a set of scripts to set up openHAB on any Debian based system (Raspbian, Ubuntu)
+*   a complete **SD-card image pre-configured with openHAB** and many other openHAB- and Hardware-specific preparations for all *Raspberry Pi* models.
 
-Closely related to openHABian is the repository [openhab-linuxpkg](https://github.com/openhab/openhab-linuxpkg) which provides linux packages for openHAB. openHABian uses those packages to setup openHAB with some additional configurations.
+## Hardware and OS support
+As of openHABian version 1.5 of the image, all Raspberry Pi models are supported as hardware.
+Anything x86 based may work or not. Anything else ARM based such as ODroids, OrangePis and the like may work or not.
+NAS servers such as QNAP and Synology boxes will not work. Support for PINEA64 was also dropped in this current release.
+We strongly recommend that users choose Raspberry Pi 2, 3 or 4 systems to have 1GB of RAM or more. RPi 1 and 0/0W only have a single CPU core and 512MB. This can be sufficient to run a smallish openHAB setup, but it will not be enough to run a full-blown system with many bindings and memory consuming openHABian features/components such as ZRAM, InfluxDB or Grafana.
+We do not actively prohibit installation on any hardware - including unsupported systems -, but we might skip or deny to install specific extensions such as those memory hungry features named above.
+Supporting hardware means testing every single patch and every release. There are simply too many SBC and HW combinations that maintainers do not have available, or, even if they did, the time to spend on the testing efforts that is required to make openHABian a reliable system.
+Let's make sure you understand the implications of these statements: it means that to run hardware other than RPi 2/3/4 or x86 is **not** _fully_ supported.
+You can try and there's a good chance it'll work out for you. But if you hit any problem related to memory or hardware, you'll be on your own. You are expected **not** to raise these problems as issues on the community forum or on GitHub.
 
-## Installation and Setup
+Going beyond what the RPi image provides, as a manually installed set of scripts, openHABian is supposed to run on all Debian like Linux distributions on either ARM or x86 hardware. Note our automated testing for x86 is based on generic Debian and we take no resposibilities if you run into trouble with a different Linux flavour.
+We recommend to use Ubuntu for x86 hardware and ARMbian for ARM hardware that we don't support.
```

Comment:
I cannot recommend Ubuntu. I am a Debian user. New Ubuntu release always bring the more recent software versions, which gave us some headaches the last weeks (Python3, etc.)
My recommendation would be clearly Debian.

Reply:
I understand you, I feel the same.
But the challenge to maintainership is we need to support what people use, not what we like best. Remember Travis testing is also based on Ubuntu, not Raspbian.

Reply:
I changed that to make clear Ubuntu is not supported for the time being.

I think we will need to support it in the future, though. So many more to use it than Debian.

Reply:
I agree, I think it will be a large benefit to us if we offer Ubuntu support in addition to Debian as Ubuntu is much more widely used.